packages/compass-e2e-tests/.npmrc

```diff
@@ -0,0 +1,4 @@
+engine-strict=true
```

Comment:
Was this commited by mistake?

Reply:
I deliberately copied .npmrc over so I could run commands in packages/compass-e2e-tests. Or do you mean that line specifically?

Reply:
(I probably only need napi-build-version=7)

Reply:
Yeah, I mean the whole file, this might create problems for people who don't expect that subpackage has a separate set of npm configurations. Why do we need it here? Can we have the same at the root?

Reply:
I'll remove it. Can always just set an env var locally and forget about it.

Reply:
Ah, wait, I think I didn't get it right the first time. So you like literally do something like `cd packages/compass-e2e-tests` and run commands from there, that's the use-case for having this `.npmrc` in the workspace directly? I'd still prefer we don't have copy-pasted `.npmrc` files so that they don't do out of sync. If it's not convenient to run commands for subpackages with lerna or `--workspace` arg, I think it's fine if you would want to add some new commands to the root to make your life easier 😃